spec/MediatorManager+services.yaml

```diff
@@ -1,293 +1,296 @@
 servers:
-## TODO: Search and replace all "xx" in the uuid by the official abbreviation of the ApplicationName
 
   operation-servers:
     own-oam:
       basic:
         - operation-name: /v1/register-yourself
-          uuid: xx-1-0-0-op-s-bm-000
+          uuid: mm-1-0-0-op-s-bm-000
         - operation-name: /v1/embed-yourself
-          uuid: xx-1-0-0-op-s-bm-001
+          uuid: mm-1-0-0-op-s-bm-001
         - operation-name: /v1/redirect-service-request-information
-          uuid: xx-1-0-0-op-s-bm-002
+          uuid: mm-1-0-0-op-s-bm-002
         - operation-name: /v1/redirect-oam-request-information
-          uuid: xx-1-0-0-op-s-bm-003
+          uuid: mm-1-0-0-op-s-bm-003
         - operation-name: /v1/end-subscription
-          uuid: xx-1-0-0-op-s-bm-004
+          uuid: mm-1-0-0-op-s-bm-004
         - operation-name: /v1/inquire-oam-request-approvals
-          uuid: xx-1-0-0-op-s-bm-005
+          uuid: mm-1-0-0-op-s-bm-005
         - operation-name: /v1/update-client
-          uuid: xx-1-0-0-op-s-bm-007
+          uuid: mm-1-0-0-op-s-bm-007
         - operation-name: /v1/list-ltps-and-fcs
-          uuid: xx-1-0-0-op-s-bm-008
+          uuid: mm-1-0-0-op-s-bm-008
         - operation-name: /v1/redirect-topology-change-information
-          uuid: xx-1-0-0-op-s-bm-009
+          uuid: mm-1-0-0-op-s-bm-009
         - operation-name: /v1/update-operation-key
-          uuid: xx-1-0-0-op-s-bm-010
+          uuid: mm-1-0-0-op-s-bm-010
         - operation-name: /v1/update-operation-client
-          uuid: xx-1-0-0-op-s-bm-011
+          uuid: mm-1-0-0-op-s-bm-011
       individual:
         - operation-name: /v1/bequeath-your-data-and-die
-          uuid: xx-1-0-0-op-s-im-000
+          uuid: mm-1-0-0-op-s-im-000
     service:
       basic:
         - operation-name: /v1/start-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-000
+          uuid: mm-1-0-0-op-s-bs-000
         - operation-name: /v1/inform-about-application
-          uuid: xx-1-0-0-op-s-bs-001
+          uuid: mm-1-0-0-op-s-bs-001
         - operation-name: /v1/inform-about-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-002
+          uuid: mm-1-0-0-op-s-bs-002
         - operation-name: /v1/inform-about-release-history
-          uuid: xx-1-0-0-op-s-bs-003
+          uuid: mm-1-0-0-op-s-bs-003
         - operation-name: /v1/inform-about-release-history-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-004
+          uuid: mm-1-0-0-op-s-bs-004
         - operation-name: /docs
-          uuid: xx-1-0-0-op-s-bs-005
+          uuid: mm-1-0-0-op-s-bs-005
       individual:
-        ## TODO: Add Services specific to this application
```

Comment:
not needed here, but if it would be helpful to you, you can add a comment for each service on what's the input and/or output (example: MATR)